ts/src/task_force_radio.cpp

```diff
@@ -111,19 +111,22 @@ TFAR::TFAR() {
         diag << TS_INDENT << TS_INDENT << "pttDelay: " << m_gameData.pttDelay.count() << "ms\n";
 
         std::array<char*, 3> stereoModeToString{ "stereo","left","right" };
+		std::array<char*, 2> halfDuplexOverrideToString{ "disabled", "enabled" };
```

Comment:
I'll check my VS settings and make sure it uses spaces, sorry about that. Will reformat properly.